addons/blue_secret_controller_test.go

```diff
@@ -96,6 +82,11 @@ func TestBlueSecretReconciler_Reconcile(t *testing.T) {
 		t.Error("failed to add rookv1 scheme")
 	}
 
+	err = storagev1.AddToScheme((scheme))
```

Comment:
These changes are probably part of ReplicationID changes. It shouldn't be in this PR.

Reply:
good catch. removed